cetmix_tower_server/models/cx_tower_key.py

```diff
@@ -50,11 +51,34 @@ class CxTowerKey(models.Model):
     _sql_constraints = [
         (
             "reference_unique",
-            "UNIQUE(reference, partner_id, server_id)",
-            "Reference must be unique",
+            "check(1=1)",
```

Comment:
why not delete this constraint in migration script?

```sql
SELECT *
FROM information_schema.constraint_table_usage
WHERE table_name = 'cx_tower_key';
ALTER TABLE cx_tower_key DROP CONSTRAINT cx_tower_key_reference_unique;
```

P.S. not tested

Reply:
This constraint is [defined](https://github.com/cetmix/cetmix-tower/blob/f62ffb248ba7e405e433f6b11d0f233581301946/cetmix_tower_server/models/cx_tower_reference_mixin.py#L29) at the mixin level, so it will be created by mixin. 
Probable solution can be deletion of this constraint in the post-init hook for new installations and in the migration script for existing ones.
@GabbasovDinar your thoughts?